packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.styles.ts

```diff
@@ -66,6 +68,33 @@ export const getOptionStyles = memoizeFunction(
     const ComboBoxOptionTextColorDisabled = semanticColors.disabledText;
     const ComboBoxOptionBackgroundDisabled = semanticColors.bodyBackground;
 
+    /**
+     * This is the same as the output from getFocusStyle(),
+     * but without `:focus` since ComboBox options don't receive focus.
+     */
+    const ComboBoxFocusStyle: IRawStyle = {
+      outline: 'transparent',
+      position: 'relative',
+
+      selectors: {
+        '::-moz-focus-inner': {
+          border: '0'
+        },
+
+        [`.${IsFocusVisibleClassName} &:after`]: {
+          content: '""',
+          position: 'absolute',
+          left: 1,
+          top: 1,
+          bottom: 1,
+          right: 1,
+          border: `1px solid ${palette.white}`,
+          outline: `1px solid ${palette.neutralSecondary}`,
+          zIndex: ZIndexes.FocusStyle

```

Comment:
Is there a way to consolidate this? If the style is exactly the same and the only difference is `:focus`, I think it'd be better to centralize and would improve ease of maintenance...

Reply:
I was thinking of making a new option in the getFocusStyle function. Something like `isFocusedOnly` or something. Thoughts?

Reply:
```ts
/**
 * Generates a focus style which can be used to define an :after focus border.
 *
 * @param theme - The theme object to use.
 * @param inset - The number of pixels to inset the border.
 * @param position - The positioning applied to the container. Must
 * be 'relative' or 'absolute' so that the focus border can live around it.
 * @param highContrastStyle - Style for high contrast mode.
 * @param borderColor - Color of the border.
 * @param outlineColor - Color of the outline.
 * @param isFocusedOnly - If the styles should apply on focus or not.
 * @returns The style object.
 */
export function getFocusStyle(
  theme: ITheme,
  inset: number = 0,
  position: 'relative' | 'absolute' = 'relative',
  highContrastStyle: IRawStyle | undefined = undefined,
  borderColor: string = theme.palette.white,
  outlineColor: string = theme.palette.neutralSecondary,
  isFocusedOnly: boolean = true
): IRawStyle {
  return {
    outline: 'transparent',
    position,

    selectors: {
      '::-moz-focus-inner': {
        border: '0'
      },

      [`.${IsFocusVisibleClassName} &${isFocusedOnly ? ':focus' : ''}:after`]: {
        content: '""',
        position: 'absolute',
        left: inset + 1,
        top: inset + 1,
        bottom: inset + 1,
        right: inset + 1,
        border: '1px solid ' + borderColor,
        outline: '1px solid ' + outlineColor,
        zIndex: ZIndexes.FocusStyle,
        selectors: {
          [HighContrastSelector]: highContrastStyle
        }
      }
    }
  };
}
```